### PR TITLE
Benchmarking Scripts

### PR DESCRIPTION
**Pull request by dlangbe**
Added scripts to perform benchmarking of all hipTensor test types

### CODE REVIEW

**Review by cgmillette — APPROVED** [review]
LGTM

**Review by CongMa13 — APPROVED** [review]
lgtm

File: scripts/performance/BenchmarkContraction.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# Copyright (C) 2022-2024 Advanced Micro Devices, Inc. All rights reserved.
+
+set -eux
+
+# ensure this script is in the cwd
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+output_dir=hiptensor-benchmarks
+build_dir=../../build/bin/
+config_dir=../../test/01_contraction/configs
```
Comment (CongMa13) [inline scripts/performance/BenchmarkContraction.sh:11]: Just found that there are some new files added in another commit

`test/01_contraction/configs/bench/bilinear_test_params_rank1.yaml`

Are they going to be used in benchmark scripts?

Comment (cgmillette) [reply scripts/performance/BenchmarkContraction.sh:11]: Yes. Benchmarks probably won't cover so much parameter space, but have more emphasis on performance and regression checks.

**Comment by cgmillette** [comment]
Superseded by #276

**Pull request closed without merging**